bootstrap.sh

```
@@ -13,6 +13,9 @@ kubectl apply -f .infrastructure/app/configMap.yml
 kubectl apply -f .infrastructure/app/clusterIp.yml
 kubectl apply -f .infrastructure/app/nodeport.yml
 kubectl apply -f .infrastructure/app/hpa.yml
+
```

This script doesn't create the kind cluster using the required cluster.yml. Checklist item #2 requires you to use kind with a cluster.yml configuration. Consider adding commands near the start to create the cluster, e.g.: `kind create cluster --config cluster.yml` (and ensure `cluster.yml` exists in the repo). Also include `set -e` at the top to fail fast on error.